### PR TITLE
Tower integration

### PR DESCRIPTION
**Pull request by naofireblade**
Tower tiers with unique properties and assets are loaded from a tower specific folder.

### CODE REVIEW

**Comment by naofireblade** [comment]
Solves #18

**Comment by naofireblade** [comment]
Reopened with TowerManager class to get tower type by name instead of index.

File: assets/js/towermanager.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		        key: 'Cannon'
+		    }
+		];
+
+		this.towerTypes.forEach(tower => {
+			this.game.load.json(tower.key + '_properties', 'assets/towers/' + tower.key + '/properties.json');
+		});
+	}
+
+	load() {
```
Comment (Milchreis) [inline assets/js/towermanager.js:17]: Formatting is not correct.

Comment (naofireblade) [reply assets/js/towermanager.js:17]: Thanks, I have to switch sublime to spaces instead of tabs for this project :)

**Pull request merged**